docs/CLERK_MIGRATION_SUMMARY.md

```
@@ -0,0 +1,413 @@
+- UUID-based user IDs
+- `auth.uid()` for RLS policies
+
+**After:**
+- Clerk for user authentication
+- TEXT-based user IDs (e.g., `user_2xxxClerkID123`)
+- `current_user_id()` custom function for RLS policies
+- Guest sessions using `guest-{sessionId}` pattern
+
+### 2. Database Schema
+
+#### New Function Added
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)